### ISSUE TITLE
Custom Standing Doors

### ISSUE THREAD

**Issue by Chee5eman** [BEEmod/BEE2.4#1865]
### Description of the feature

I know its a stretch but I think adding custom standing doors could be a cool edition to Portal 2 maps using **Bee2**.

**What do I mean by "Custom standing doors"?**

_well_ what I'm talking about is the texture of the standing door... and how you can change the color of the door "it will use the same color picker as the cube color selector in the (style/item properties menu)" or be able to fully change the texture of the door _"but I know that's a stretch"_

### Why should this be implemented

i think it could be add some extra customization of **bee2** amazing tools... and give maps using **bee2** mod some added life...

### Additional information

Here are some examples of what I mean...

![paintable doors](https://user-images.githubusercontent.com/111412878/185049516-9146c83d-9f91-4a6b-ade4-c90ba82dd94b.png)

**Comment by HeimlichLaboratories** [comment]
Correct me if I'm wrong but doesn't tinting a prop tint the whole model?

**Comment by FrozenDragon0** [comment]
> Correct me if I'm wrong but doesn't tinting a prop tint the whole model?

If they aren't setup for this type of thing then yes, the cube colorizer has a custom texture that's set up to allow the colors to be applied, doing this would require a custom texture for the door.

**Comment by TeamSpen210** [comment]
What purpose would this serve? The reason colourised cubes were added is not for aesthetics - it's to allow matching cubes to their dropper, in puzzles where cube recall is relevant. Doors are fixed in place, and you really shouldn't have too many of them. Note that Valve's convention is that the testchamber door should *only* be used to separate different puzzles, with panel-arm doors used instead in the middle of a puzzle.

**Comment by HeimlichLaboratories** [comment]
Yeah, BEE2 is for puzzles, not for visuals (with the exception of styles, and those are major changes anyway)

**Comment by LautaroL20** [comment]
You have posted in the wrong section, anything related to items or styles must be posted on the items repository, here: https://github.com/BEEmod/BEE2-items/issues

The section where you have posted is for things related to the BEEmod user interface, only.
